docs/roles/system_manager/manage_deployments.md

```diff
@@ -6,6 +6,34 @@
 This document assumes that you already have access to a {ref}`Safe Haven Management (SHM) environment <deploy_shm>` and one or more {ref}`Secure Research Environments (SREs) <deploy_sre>` that are linked to it.
 ```
 
+(resize_vm)=
+
+## {{arrow_upper_right}} Resize the Virtual Machine (VM) of a Secure Research Desktop (SRD)
+
+Sometimes during a project that uses a deployed SRE, researchers may find the available compute inadequate for their purposes and wish to increase the size of the SRD's VM.
+
+- Log into the Azure portal and locate the VM inside the Resource Group called `RG_SHM_<shm id>_SRE_<sre id>_COMPUTE`
+- The simplest way to resize this VM is by [following these instructions](https://learn.microsoft.com/en-us/azure/virtual-machines/resize-vm?tabs=portal) in the Azure portal
+- Alternatively, you can resize this VM by using the below instructions to "Add a new SRD", taking care to set the `<IP last octet>` to that of the VM (viewable in the Azure portal), and by setting the `-Upgrade` and `-Force` flags
+
+## {{heavy_plus_sign}} Add a new SRD
+
+The `-VmSizes` parameter provided when deploying the SRE (with the `Deploy_SRE.ps1` script) determines how many SRDs are created and how large each one will be.
+
+To deploy a new SRD into the SRE environment, follow the below instructions:
+
+![Powershell: ten minutes](https://img.shields.io/static/v1?style=for-the-badge&logo=powershell&label=local&color=blue&message=ten%20minutes) at {{file_folder}} `./deployment/secure_research_environment/setup`
+
+```powershell
+PS> ./Add_Single_SRD.ps1 -shmId <SHM ID> -sreId <SRE ID> -ipLastOctet <IP last octet> [-vmSize <VM size>]
+```
+
+- where `<SHM ID>` is the {ref}`management environment ID <roles_deployer_shm_id>` for this SHM
+- where `<SRE ID>` is the {ref}`secure research environment ID <roles_deployer_sre_id>` for this SRE
+- where `<IP last octet>` is last octet of the IP address (if resizing an existing VM, check what this is in Azure)
+- [optional] where `<VM size>` is the [Azure VM size](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes) for this SRD
+- [optional] where resizing a VM for an existing SRD, also add `-Upgrade` and `-Force`
```

Comment:
With the above change, this line would then be unnecessary.

I think it risks confusing readers to talk about resizing an existing SRD in the section about creating a new one.